### ISSUE TITLE
what is the meaning of dev in this ?

### ISSUE THREAD

**Issue by cameronbergh** [wangcongcong123/auto_coding#5]
in most deep learning projects that i have seen, the data is split into training and testing. but here it seems to be split into training and dev. i am not familiar with this, at first i thought it may have been a translation error but i see in evaluate.py that it is coded explicitly on line 19

 `if data_tag not in ["dev", "train", "test"]:`

maybe it is train, test, and validation?

**Comment by cameronbergh** [comment]
oh i found it. (https://en.wikipedia.org/wiki/Training,_validation,_and_test_sets)

`The terms test set and validation set are sometimes used in a way that flips their meaning in both industry and academia. In the erroneous usage, "test set" becomes the development set, and "validation set" is the independent set used to evaluate the performance of a fully specified classifier.`
